playbooks/roles/capsule/tasks/install.yml

```diff
@@ -31,7 +31,7 @@
   register: pulp_oauth_secret
 
 - name: 'Generate Certs'
-  command: capsule-certs-generate --capsule-fqdn {{ ansible_nodename }} --certs-tar {{ capsule_certs_tar }} --disable-system-checks
+  command: capsule-certs-generate --capsule-fqdn {{ ansible_nodename }} --certs-tar {{ capsule_certs_tar }}
```

Comment:
This really fixes it? Doesn't capsule-cert-generate still use our hooks directory?
https://github.com/Katello/katello-installer/blob/master/bin/capsule-certs-generate#L52